.travis-build-without-test.sh

```diff
@@ -32,5 +37,20 @@ echo "Running:  (cd ../annotation-tools/ && ./.travis-build-without-test.sh)"
 echo "... done: (cd ../annotation-tools/ && ./.travis-build-without-test.sh)"
 
 ## Compile
-echo "running \"ant dist\" for checker-framework"
-ant dist
+#TODO: The docker images should be updated to have wget installed.
```

Comment:
This will be complete in about 1.5 hours, so the comment and the two lines of code can be removed.